src/App.js

```diff
@@ -75,72 +74,37 @@ function App() {
     setPlayer("O");
   };
 
-  return (
-    <div className="App">
-      <div className="board">
-        <div className="row">
-          <Square
-            val={board[0]}
-            chooseSquare={() => {
-              chooseSquare(0);
-            }}
-          />
-          <Square
-            val={board[1]}
-            chooseSquare={() => {
-              chooseSquare(1);
-            }}
-          />
-          <Square
-            val={board[2]}
-            chooseSquare={() => {
-              chooseSquare(2);
-            }}
-          />
-        </div>
-        <div className="row">
-          <Square
-            val={board[3]}
-            chooseSquare={() => {
-              chooseSquare(3);
-            }}
-          />
-          <Square
-            val={board[4]}
-            chooseSquare={() => {
-              chooseSquare(4);
-            }}
-          />
-          <Square
-            val={board[5]}
-            chooseSquare={() => {
-              chooseSquare(5);
-            }}
-          />
-        </div>
-        <div className="row">
-          <Square
-            val={board[6]}
-            chooseSquare={() => {
-              chooseSquare(6);
-            }}
-          />
-          <Square
-            val={board[7]}
-            chooseSquare={() => {
-              chooseSquare(7);
-            }}
-          />
+    // Use two loops to make the squares
```

Comment:
Read https://itnext.io/what-makes-a-good-code-comment-5267debd2c24

Reply:
Thanks for the article, a great read!